docs/source/installation/for_dev.rst

```diff
@@ -0,0 +1,126 @@
+Manim Installation For Developers
+=================================
+
+	This documentation is for developers who wats to contribute to ``ManimCommunity/manim``.
+
+Installing Poetry
+*****************
+
+Poetry can be easily installed in any OS by just running the below command.
+
+.. code-block:: bash
+	
+	curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+
+
+If you are a Windows developer and want to use PowerShell, you can use the below command.
```

Comment:
Issue: the previous line said that you can use `curl` on _any_ system. But this line is saying something different. This seems contradictory.

Suggestion: change the previous line to something like "if your system has the `curl` program, then you can write the following command. `curl` is usually installed on linux and Mac OS". The next line should say "in case you are on Windows/Powershell, blah blah blah "